community/membership.md

```diff
@@ -0,0 +1,286 @@
+| lead         | set direction and priorities for a subproject | demonstrated responsibility and excellent technical judgement for the subproject | Member of Argo TOC |
+
+
+
+## New contributors
+
+New contributors should be welcomed to the community by existing members,
+helped with PR workflow, and directed to relevant documentation and
+communication channels.
+
+
+
+## Established community members
+
+Established community members are expected to demonstrate their adherence to the
+principles in this document, familiarity with project organization, roles,
+policies, procedures, conventions, etc., and technical and/or writing ability.
+Role-specific expectations, responsibilities, and requirements are enumerated
+below.
+
+
+
+## Member
+
+Members are continuously active contributors in the community. They can have issues and PRs 
+assigned to them. Members are expected to remain active contributors to the community.
+
+Defined by: Member of the Argoproj GitHub organization
+
+### Requirements
+
+- Enabled [two-factor authentication] on their GitHub account
+- Have made multiple contributions to the project or community.  Contribution may include, but is not limited to:
+    - Authoring or reviewing PRs on GitHub
+    - Filing or commenting on issues on GitHub
+    - Contributing to SIG, subproject, or community discussions (e.g. meetings, Slack, email discussion
+      forums, Stack Overflow)
+- Actively contributing to 1 or more subprojects.
+- Sponsored by 2 approvers or leads. **Note the following requirements for sponsors**:
+    - Sponsors must have close interactions with the prospective member - e.g. code/design/proposal review, coordinating
+      on issues, etc.
+    - Sponsors must be approvers in at least 1 OWNERS file either in any of the four main subprojects in the [Argoproj org].
+      - An approver in the [Argoproj org] may sponsor someone for the [Argoproj org]
+      or any of the related [Argoproj GitHub organizations]; as long as it's a project they're involved with.
+      - A sponsor who is an approver in any of the related [Argoproj GitHub organizations]
+      but not in the [Argoproj org] can only sponsor someone for the orgs they are associated with.
+    - Sponsors must be from multiple member companies to demonstrate integration across community.
+- **[Open an issue][membership request] against the argoproj/argoproj repo**
+   - Ensure your sponsors are @mentioned on the issue
+   - Complete every item on the checklist ([preview the current version of the template][membership template])
+   - Make sure that the list of contributions included is representative of your work on the project.
+- Have your sponsoring reviewers reply confirmation of sponsorship: `+1`
+- Once your sponsors have responded, your request will be reviewed by the Argoproj GitHub Admin team. Any missing information will be requested.
```

Comment:
Do we document who the `Argoproj GitHub Admin team` is anywhere?

Reply:
Removed and replaced with team leads